src/chrome/content/rules/Duolingo.xml

```diff
@@ -0,0 +1,28 @@
+<!--
+	Mismatched:
+		^duolingo.cn		equal to www.duolingo.cn
+	Timeout:
+		sponsorships.duolingo.com
+-->
+<ruleset name="Duolingo.com">
+	<target host="duolingo.com" />
+	<target host="*.duolingo.com" />
+
+	<target host="duolingo.cn" />
+	<target host="www.duolingo.cn" />
+	<target host="api.duolingo.cn" />
+	<target host="englishtest.duolingo.cn" />
+
+	<securecookie host=".+" name=".+" />
+
+	<exclusion pattern="^http://sponsorships\.duolingo\.com/" />
+		<test url="http://sponsorships.duolingo.com/" />
+		<test url="http://www.duolingo.com/" />
+		<test url="http://api.duolingo.com/" />
+		<test url="http://englishtest.duolingo.com/" />
+		<test url="http://de.duolingo.com/" />
```

Comment:
Can you do the opposite, e.g. only include working targets.

Reply:
are there tools to find subdomains all?

Reply:
@ivysrono https://github.com/OWASP/Amass